LiteCore/Query/QueryParser.cc

```diff
@@ -745,12 +745,14 @@ namespace litecore {
 
     static string columnTitleFromProperty(const Path &property, bool useAlias) {
         if (property.empty())
-            return "*";
+            return "*"; // for the property ".", i.e. the entire doc
```

Comment:
This should be eventually translated to the name of the data source (i.e. `select * from orders` should be `orders` not `*`)  Has this been checked?

Reply:
Not yet. I thought about it but balked for this talked. It is little tricky for Mobile N1QL, because we don't require the from clause. The default is "_doc", little ugly. Perhaps it could "default," or the like. Perhaps a separate ticket?

Reply:
I thought we decided to make the FROM clause mandatory.  It cannot just be "default", it should mirror what server does.

Reply:
Not reflected in the code yet. This will invalidate many test cases.

Reply:
Especially important to do this once we support collections.
For now on master, add the default alias.

Reply:
Got it. Doing it now.